src/main/java/org/broadinstitute/hellbender/utils/test/testers/AbstractMarkDuplicatesTester.java

```diff
@@ -19,6 +19,9 @@
  * This class is an extension of SamFileTester used to test AbstractMarkDuplicatesCommandLineProgram's with SAM files generated on the fly.
  * This performs the underlying tests defined by classes such as AbstractMarkDuplicatesCommandLineProgramTest.
  */
+// TODO: is this really necessary for the packaged testing framework?
+// TODO: it looks like that this is a SamFileTester exclusive for MarkDuplicates with the parameter in GATK/Picard
+// TODO: and thus, it should live in the test resources
 public abstract class AbstractMarkDuplicatesTester extends SamFileTester {
```

Comment:
I agree, though I would leave it for a separate pull request to make that change for the testers more broadly.

Reply:
Changed the TODO to a clear procedure and open a new ticket for move it to the test sources (#3762)